scripts/run

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 
 const path = require('path');
-const tsConfigPath = path.resolve(__dirname, 'tsconfig.base.json');
+const tsConfigPath = path.resolve(__dirname, 'tsconfig.json');
```

Comment:
you should not be using tsconfig.json for now until Angular 10.1.0 (where they rollback the solution style configs). Right now tsconfig is for IDEs and tooling, as it contains all references.

Reply:
This tsconfig is only used for the ts-node compiler, it is not used anywhere else. The 'npm run lite-serve' script was errored because of it. I've just restored it, now it works.